docs/install/update-harvester-configuration.md

```diff
@@ -0,0 +1,253 @@
+
+:::note
+
+If you upgrade from a version before `v1.1.2`, the `cloud-init` file in examples will be `/oem/99_custom.yaml`. Please substitute the value if needed.
+
+:::
+
+## DNS servers
+
+### Runtime change
+
+1. Log in to a Harvester node and become root. See [how to log into a Harvester node](../troubleshooting/os.md#how-to-log-into-a-harvester-node) for more details.
+1. Edit `/etc/sysconfig/network/config` and update the following line. Use a space to separate DNS server addresses if there are multiple servers.
+
+    ```
+    NETCONFIG_DNS_STATIC_SERVERS="8.8.8.8 1.1.1.1"
+    ```
+
+1. Update and reload the configuration with the following command:
+
+    ```
+    netconfig update
+    ```
+
+1. Confirm the file `/etc/resolv.conf` contains the correct DNS servers with the `cat` command:
+
+    ```
+    cat /etc/resolv.conf
+    ```
+
+### Configuration persistence
+
+1. Backup the elemental `cloud-init` file `/oem/90_custom.yaml` as follows:
+
+    ```
+    cp /oem/90_custom.yaml /oem/install/90_custom.yaml.$(date --iso-8601=minutes)
+    ```
+
+1. Edit `/oem/90_custom.yaml` and update the value under the yaml path `stages.initramfs[0].commands`. The `commands` array must contain a line to manipulate the `NETCONFIG_DNS_STATIC_SERVERS` config. Add the line if the line doesn't exist. 
+
+The following example adds a line to change the `NETCONFIG_DNS_STATIC_SERVERS` config:
+
+    ```
+    stages:
+      initramfs:
+        - commands:
+            - sed -i 's/^NETCONFIG_DNS_STATIC_SERVERS.*/NETCONFIG_DNS_STATIC_SERVERS="8.8.8.8 1.1.1.1"/' /etc/sysconfig/network/config
+    ```
+
+    Replace the DNS server addresses and save the file. Harvester sets up new servers after rebooting.
+
+
+## NTP servers
+
+### Runtime change
+
+1. Log in to a Harvester node and become root. See [how to log into a Harvester node](../troubleshooting/os.md#how-to-log-into-a-harvester-node) for more details.
+1. Edit `/etc/systemd/timesyncd.conf` and specify NTP servers in the `NTP=` setting:
+
+    ```
+    [Time]
+    NTP = 0.suse.pool.ntp.org 1.suse.pool.ntp.org
+    ```
+
+1. Restart the `systemd-timesyncd.service` service:
+
+    ```
+    systemctl restart systemd-timesyncd.service
+    ```
+
+1. Display the timesync status:
+
+    ```
+    timedatectl timesync-status
+    ```
+
+### Configuration persistence
+
+1. Backup the elemental `cloud-init` file `/oem/90_custom.yaml` as follows:
+
+    ```
+    cp /oem/90_custom.yaml /oem/install/90_custom.yaml.$(date --iso-8601=minutes)
+    ```
+
+1. Edit `/oem/90_custom.yaml` and update the yaml path `stages.initramfs[0].timesyncd`. The `timesyncd` map must be in the following format:
+
+    ```
+    stages:
+      initramfs:
+      - ...
+        timesyncd:
+        NTP: 0.suse.pool.ntp.org 1.suse.pool.ntp.org
+    ```
+
+1. Edit `/oem/90_custom.yaml` and update the yaml path `stages.initramfs[0].systemctl.enable`. The array must have these two services enabled: 
```

Comment:
```suggestion
1. Edit `/oem/90_custom.yaml` and update the yaml path `stages.initramfs[0].systemctl.enable`. The array must have these two services (`systemd-timesyncd` and `systemd-time-wait-sync`) enabled: 
```

Reply:
Could we specify these two services more?

Reply:
Fixed!